reference/sql/statements/flashback-table.md

```diff
@@ -0,0 +1,84 @@
+
+Use the following command to query the TiDB cluster's `tikv_gc_safe_point` and `tikv_gc_life_time`. As long as the table is dropped by `DROP` or `TRUNCATE` after the `tikv_gc_safe_point` time, you can restored the table using the `FLASHBACK TABLE` statement.
+
+{{< copyable "sql" >}}
+
+  ```sql
+  select * from mysql.tidb where variable_name in ('tikv_gc_safe_point','tikv_gc_life_time');
+  ```
+
+## Syntax
+
+{{< copyable "sql" >}}
+
+```sql
+FLASHBACK TABLE table_name [TO other_table_name]
+```
+
+## Notes
+
+If a table is dropped and the GC lifetime has passed, you can no longer use the `FLASHBACK TABLE` statement to recover the dropped data. Otherwise, an error will be returned, which is similar to `Can't find dropped / truncated table 't' in GC safe point 2020-03-16 16:34:52 +0800 CST`.
+
+Pay attention to the following conditions and requirements when you enable TiDB Binlog and use the `FLASHBACK TABLE` statement:
+
+* The downstream cluster must also support `FLASHBACK TABLE`.
+* The GC lifetime of the slave cluster must be longer than that of the master cluster.
+* The delay of replication between the upstream and downstream might also cause the failure to recover data to the downstream.
+* If an error occurs when TiDB Binlog is replicating a table, you need filter that table in TiDB Binlog and manually import all data of that table.
+
+## Example
+
+- Recover the table data dropped by `DROP`:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    DROP TABLE t;
+    ```
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    FLASHBACK TABLE t;
+    ```
+
+- Recover the table data dropped by `TRUNCATE`. Because the truncated table `t` still exists, you need to rename the table `t` to be recovered. Otherwise, an error will be returned indicating that table `t` already exists.
+
+    ```sql
+    TRUNCATE TABLE t;
+    ```
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    FLASHBACK TABLE t TO t1;
+    ```
+
+## Implementation principle
+
+When deleting a table, TiDB only deletes the table metadata, and writes the table data (row data and index data) to be deleted to the `mysql.gc_delete_range` table. The GC Worker in the TiDB background periodically removes from the `mysql.gc_delete_range` table the keys that exceed the GC lifetime.
+
+Therefore, to recover a table, you only need to recover the table metadata and delete the corresponding row record in the `mysql.gc_delete_range` table before the GC Worker deletes the table data. You can use a snapshot read of TiDB to recover the table metadata. Refer to [Read Historical Data](/how-to/get-started/read-historical-data.md) for details. The following is the working process of `FLASHBACK TABLE t TO t1`:
+
+1. TiDB searches the recent DDL job history and locates the first DDL operation of the `DROP TABLE` type. If TiDB fails to locate one, an error is returned.
```

Comment:
This sentence is not aligned with the Chinese version. Please update it. @TomShawn